packages/react-components/lib/theme/theme.tsx

```diff
@@ -0,0 +1,52 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+declare module '@material-ui/core/styles/createMuiTheme' {
+  interface Theme {
+    colors: {
+      black: React.CSSProperties['color'];
+      xanadu: React.CSSProperties['color'];
+      stPatricksBlue: React.CSSProperties['color'];
+      blueYonder: React.CSSProperties['color'];
+      ghostWhite: React.CSSProperties['color'];
+      indianYellow: React.CSSProperties['color'];
+      fireOpal: React.CSSProperties['color'];
+    };
+  }
+  interface ThemeOptions {
+    colors: {
+      black: React.CSSProperties['color'];
+      xanadu: React.CSSProperties['color'];
+      stPatricksBlue: React.CSSProperties['color'];
+      blueYonder: React.CSSProperties['color'];
+      ghostWhite: React.CSSProperties['color'];
+      indianYellow: React.CSSProperties['color'];
+      fireOpal: React.CSSProperties['color'];
+    };
+  }
+}
+
+export const customTheme = createMuiTheme({
+  palette: {
+    primary: {
+      // Dark cornflower blue
+      main: '#1F396B',
+      light: '#4B6088',
+      dark: '#15274A',
+    },
+    secondary: {
+      // Light sea green
+      main: '#20A39E',
+      light: '#4CB5B1',
+      dark: '#16726E',
+    },
+  },
+  colors: {
+    black: '#000000',
+    xanadu: '#6B7D7D',
+    stPatricksBlue: '#103375',
+    blueYonder: '#5873A8',
+    ghostWhite: '#FBFCFF',
+    indianYellow: '#DEA54B',
+    fireOpal: '#F25F5C',
```

Comment:
Are they any design specs for these colors? They are not part of the palette.

Reply:
I am following the color scheme provided in figma:
https://www.figma.com/file/nFiXHduImJev6xTodDpd0K/TNG-WIP?node-id=3%3A5814

Reply:
These random colors without any design specs and not in the palette are very confusing. How do I know if/when I should use these colors vs the ones in the palette?